src/graphql/mutations.js

```diff
@@ -110,14 +110,20 @@ export const SET_REGISTRANT = gql`
 `
 
 export const COMMIT = gql`
-  mutation commit($label: String, $owner: String, $secret: String) {
-    commit(label: $label, owner: $owner, secret: $secret) @client
+  mutation commit($label: String) {
+    commit(label: $label) @client
   }
 `
 
 export const REGISTER = gql`
   mutation register($label: String, $duration: Int) {
-    register(label: $label, duration: $duration, secret: $secret) @client
+    register(label: $label, duration: $duration) @client
```

Comment:
Why do you take out secret when registrar.reister still takes it as an argument ? ub.com/ensdomains/ens-app/blob/renew/src/api/registrar.js#L295

Reply:
It takes from inside the resolver, not from the front-end, so we don't need it in the mutation